src/main/java/seedu/address/logic/commands/ListCommand.java

```diff
@@ -18,6 +19,7 @@
     public CommandResult execute(Model model) {
         requireNonNull(model);
         model.sortFilteredPersonList();
+        model.updateFilteredPersonList(PREDICATE_SHOW_ALL_PERSONS);
```

Comment:
now the list of persons to be displayed is updated!

Reply:
Oh so that's what this line is supposed to do... Thanks for the explanation! :smiley: